spec/classes/config_spec.rb

```diff
@@ -79,4 +79,38 @@
       end
     end
   end
+
+  describe '#envvars' do
+    context 'valid hash' do
+      let(:params) {{
+        :envvars => {
+          'GODEBUG' => '"netdns=go"',
```

Comment:
Is it worth removing the quotes from the environment variable here and instead wrapping the `#{v}` in quotes below? That means that env vars with spaces will work.